cockatrice/src/dlg_creategame.cpp

```diff
@@ -157,8 +175,41 @@ DlgCreateGame::DlgCreateGame(const ServerInfo_Game &gameInfo, const QMap<int, QS
     setWindowTitle(tr("Game information"));
 }
 
+void DlgCreateGame::actReset()
+{
+    descriptionEdit->setText("");
+    maxPlayersEdit->setValue(2);
+
+    passwordEdit->setText("");
+    onlyBuddiesCheckBox->setChecked(false);
+    onlyRegisteredCheckBox->setChecked(true);
+
+    spectatorsAllowedCheckBox->setChecked(true);
+    spectatorsNeedPasswordCheckBox->setChecked(false);
+    spectatorsCanTalkCheckBox->setChecked(false);
+    spectatorsSeeEverythingCheckBox->setChecked(false);
+
+    QMapIterator<int, QCheckBox *> gameTypeCheckBoxIterator(gameTypeCheckBoxes);
+    while (gameTypeCheckBoxIterator.hasNext()) {
+        gameTypeCheckBoxIterator.next();
+        gameTypeCheckBoxIterator.value()->setChecked(false);
+    }
+
+    descriptionEdit->setFocus();
+}
+
 void DlgCreateGame::actOK()
 {
+    settingsCache->setGameDescription(descriptionEdit->text());
+    settingsCache->setMaxPlayers(maxPlayersEdit->value());
+    settingsCache->setGamePassword(passwordEdit->text());
```

Comment:
Re-thinking this, you're partially right. It doesn't really matter that the game passwords are stored as plain text (as they have no significance to anything other then who could join a game), but I don't see a reason a user would/should use the same PW for each and every game he's making.